Blackjack.Tests/HandShould.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using Xunit;
+
+namespace Blackjack.Tests
+{
+    public class HandShould // TODO: should i be testing anything else?

```

Comment:
rule of thumb: test all public functions and have test cases for any logic/branching within each function, e.g. if your function has if-else then you need to test the 2 branches.
In your test case below, what I'm seeing is a test for the AddCard function - i.e. that calling add will add one card to the hand. However, the name of the test is suggesting that when the Hand is initialised it will already have 2 cards.